File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    let splitedS = s.split(" ");
    let result = [];
    
    for ( let i = 0; i < splitedS.length; i++) {
        let transferedWord = [];
        
        for (let j = 0; j<splitedS[i].length; j++) {
            if (j === 0 ) transferedWord.push(splitedS[i][j].toUpperCase());
            else if ( j % 2 === 0) transferedWord.push(splitedS[i][j].toUpperCase());
            else transferedWord.push(splitedS[i][j].toLowerCase());
        }
        result.push(transferedWord.join(""));
    }
    
    console.log(result);
    console.log(result.join(""));
    return result.join(" ");
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    result = []
    
    for word in s.split(' '):
        transformed = ''
        for i, ch in enumerate(word):
            if i % 2 == 0:
                transformed += ch.upper()
            else:
                transformed += ch.lower()
        result.append(transformed)
    return ' '.join(result)